js-packages/@prestojs/viewmodel/src/ViewModelCache.ts

```diff
@@ -508,6 +567,8 @@ export default class ViewModelCache<T extends ViewModelInterface<any, any>> {
             recordCache = new RecordCache(this.viewModel, this.onAnyChange.bind(this));
             this.cache.set(pkKey, recordCache);
         }
+        // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+        // @ts-ignore
         recordCache.add(record);
```

Comment:
what was the complaint here?

Reply:
oops none anymore, i had an original version done differently